osac-aap/tests/integration/setup_test_env.sh

```
@@ -170,15 +175,116 @@ CSIEOF
+
+    echo "Starting local OCI registry (TLS) on port ${CSI_DRIVER_TEST_REGISTRY_PORT}..."
+    "${CSI_DRIVER_TEST_CONTAINER_TOOL}" rm -f osac-test-csi-registry > /dev/null 2>&1 || true
+    "${CSI_DRIVER_TEST_CONTAINER_TOOL}" run -d --name osac-test-csi-registry \
+      -p "${CSI_DRIVER_TEST_REGISTRY_PORT}:5000" \
+      -v "${SCRIPT_DIR}/certs:/certs:ro" \
+      -e REGISTRY_HTTP_TLS_CERTIFICATE=/certs/registry.pem \
+      -e REGISTRY_HTTP_TLS_KEY=/certs/registry.key \
+      registry:2 > /dev/null
+
+    echo "Waiting for local OCI registry to be ready..."
+    for i in $(seq 1 10); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)